test/prerna/testing/ApiSemossTestUtils.java

```diff
@@ -91,8 +91,29 @@ public static NounMetadata processRawPixel(String pixel) {
 		NounMetadata ret = pr.getResults().get(0);
 		return ret;
 	}
-
-	public static String buildPixelCall(Class<?> cl, Object... args) {
+	

```

Comment:
Allows for Pixel Chains with Select statements Database() | Select().as([]) | Iterate() | ToXml();